### ISSUE TITLE
2.0.42 breaks backup functionality

### ISSUE THREAD

**Issue by MrMette** [standardnotes/desktop#125]
I'm not sure if you are aware of this or not, but I was looking at my backups and noticed it didn't backup anymore for some days now. I tried switching to a different directory and back to see if it would fix it, but it didn't (haven't tried to log out and log in again).

After I reinstalled 2.0.41, it backups work fine again.

I didn't realize 2.0.42 was still in pre-release fase when I installed it, so maybe you are still working on it, but I wanted to create an issue here just if you weren't aware it didn't work any more.

This is on a Windows PC btw, I didn't test it on other systems.

**Comment by mobitar** [comment]
So backups are only created when the app is running, every 12 hours. If the app is not running on a given day, a backup will not be created.

You can test to make sure it's working by quitting the app, then re-opening it. It will create a backup on open. I'm using 2.0.42 and am seeing backups.

Let me know what you see.

**Comment by MrMette** [comment]
I did reopen the app before I updated to see if that would fix it. The last backup I had was from November 22nd and I am 100% sure I have reopened the app after that as I usually shut down my PC during weekdays and I have left it open for more then 1 day at a time as well.

I'm not sure why it doesn't work, but I did test some things about before creating this issue.
I will reinstall 2.0.42 and test it again.
